src/Core/Services/Implementations/StripeSyncService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+
+namespace Bit.Core.Services
+{
+    public class StripeSyncService : IStripeSyncService
+    {
+        private readonly IStripeAdapter _stripeAdapter;
+
+        public StripeSyncService(IStripeAdapter stripeAdapter)
+        {
+            _stripeAdapter = stripeAdapter;
+        }
+
+        public async Task<bool> UpdateCustomerEmailAddress(string gatewayCustomerId, string emailAddress)
```

Comment:
I'd probably get rid of the boolean return and just let the exception bubble up. The caller can try-catch and decide on how to handle the failure. In most cases a rollback of the email change. Also with the current implementation, the reason for the failure gets swallowed. What is the message returned from stripe, could/should we pass that on as IdentityResult.Failed

Reply:
stripe throws an exception if something wrong happens